test/evp_extra_test.c

```diff
@@ -2418,13 +2418,11 @@ static int test_evp_iv(int idx)
 static int test_EVP_rsa_pss_with_keygen_bits(void)
 {
     int ret;
-    OSSL_PROVIDER *provider;
     EVP_PKEY_CTX *ctx;
     EVP_PKEY *pkey;
     const EVP_MD *md;
     pkey = NULL;
```

Comment:
preexisting nit: blank line here.

Reply:
I went looking for this issue before seeing you had fixed it. Creating a libctx also works.

Reply:
Fixed. I do not think this needs reapproval.

Reply:
It doesn't IMO too.